main.c

```diff
@@ -309,6 +326,14 @@ static void processQueue(void)
           gpioWrite(LED2_PIN, 1);
         }
 
+        if(MAX77650_getCHG()){

```

Comment:
drop this inside !streaming.

no need to do this while the device is streaming in my opinion.